### PR TITLE
Update MobaXterm setup instructions

### PR DESCRIPTION
**Pull request by dhashe**
Instruct students to install nano before continuing with setup instructions

### CODE REVIEW

**Comment by anbenson** [comment]
Everywhere online seems to reference using apt-get instead of apt-cyg in MobaXTerm, and the references to apt-get are more recent than the references to apt-cyg on MobaXTerm's website. Have you tested this to make sure "apt-cyg install nano" works?

File: topics/terminal-usage/initial-setup/index.md [hunk 1]
```diff
@@ -78,6 +78,17 @@ other than Windows 10), your copy of Windows does not include this terminal. In
...
+before continuing with setup. This command will install the nano program, which
+is a simple text editor. You may need to confirm the installation by pressing y
+and then enter if prompted. You should verify that you see an [image similar to
+this][mobaxterm-startup] before running the command.
+
+{% highlight bash %}
+apt-cyg install nano
+{% endhighlight %}
+
 If that number is 14316 or greater, you should follow these instructions to
```
Comment (jbrown215) [inline topics/terminal-usage/initial-setup/index.md]: This number seems out of context now

**Review by jbrown215 — CHANGES_REQUESTED** [review]
I think you should probably move this paragraph around somewhere else (or edit the instructions a bit more).
Also letting you address Andrew first :)

File: topics/terminal-usage/initial-setup/index.md (hunk 1 above)
Comment (anbenson) [reply topics/terminal-usage/initial-setup/index.md]: 14316 is the first build of Windows 10 that includes the built-in Linux subsystem and terminal :)

https://blogs.windows.com/windowsexperience/2016/04/06/announcing-windows-10-insider-preview-build-14316/

Comment (jbrown215) [reply topics/terminal-usage/initial-setup/index.md]: Oh, I know-- I only meant it's separated too far from the time you're first told to look for it

**Comment by dhashe** [comment]
> Everywhere online seems to reference using apt-get instead of apt-cyg in MobaXTerm, and the references to apt-get are more recent than the references to apt-cyg on MobaXTerm's website. Have you tested this to make sure "apt-cyg install nano" works?

I have tested this, and "apt-cyg install nano" works. "apt-get" in MobaXTerm points to "apt-cyg". I think referring to it as "apt-get" is a bit misleading and might confuse students later on.

File: topics/terminal-usage/initial-setup/index.md (hunk 1 above)
Comment (dhashe) [reply topics/terminal-usage/initial-setup/index.md]: I think just changing the order of the Bash for Windows and MobaXTerm paragraphs should fix things. Then I can add the new paragraph at the end and everything stays in context.

**Review by jbrown215 — APPROVED** [review]
Looks great to me 

#PlanesWIthWifi

**Pull request merged**